### ISSUE TITLE
Missing SUPER troops in enums

### ISSUE THREAD

**Issue by majordoobie** [mathsman5133/coc.py#52]
Still on the old version. It is missing 3 super troops in the coc.enums.SUPER_TROOP_ORDER which causes the ordered_home_troops() to break. Fix is just to add the three troops to the list. Proof of Concept below:

```
new_troops = ["Super Archer","Inferno Dragon","Super Valkyrie","Super Witch"] 
coc.enums.SUPER_TROOP_ORDER.extend(new_troops)
coc.enums.HOME_TROOP_ORDER = coc.enums.ELIXIR_TROOP_ORDER + coc.enums.SUPER_TROOP_ORDER + coc.enums.DARK_ELIXIR_TROOP_ORDER + coc.enums.SIEGE_MACHINE_ORDER

key_order = {k: v for v, k in enumerate(coc.enums.HOME_TROOP_ORDER)}
OrderedDict(sorted(player.home_troops_dict.items(), key=lambda i: key_order.get(i[0])))
```

**Comment by majordoobie** [comment]
Idk how you are handling this in the future, but maybe to avoid this issue you can create another group of troop items that is not assigned to a home yet. Might prevent it from breaking completely every-time a new super comes out

```python
for item in player.troops:
  if item.name not in coc.enums.HOME_TROOP_ORDER and item.name not in coc.enums.BUILDER_TROOPS_ORDER:
    coc.enums.NEW_TROOPS_LIST.append(item.name)

# Then add to the list of HOME_TROOP_ORDER
```

**Comment by mathsman5133** [comment]
Thanks, fixed by 15b39bff20aa8141626151a4dfe787e65ce95fde
